.github/workflows/ci.yaml

```diff
@@ -1,66 +1,128 @@
+name: Continuous Integration
+
 on:
   push:
     branches: [main]
   pull_request:
     branches: [main]
   merge_group:
     types: [checks_requested]
-
-name: Continuous integration
+  paths-ignore:
+    - "**.md"
+    - "**.yml"
```

Comment:
be sure to add `**.yaml` here - my example was from teh stacks-core repo, where we use `.yml` as the file extension

Reply:
I'll address this in another PR. Thanks!